nion/swift/ExportDialog.py

```diff
@@ -261,18 +261,18 @@ class UnitType(enum.Enum):
 
 
 class ExportSizeModel(Observable.Observable):
-    def __init__(self, display_item: DisplayItem.DisplayItem, ui: UserInterface.UserInterface) -> None:
+    def __init__(self, display_item: DisplayItem.DisplayItem, units_persistent_model: UserInterface.StringPersistentModel) -> None:
```

Comment:
In this case, you can just call it `units_model: Model.PropertyModel[str]` instead of the more specific `units_persistent_model: UserInterface.StringPersistentModel`. When receiving parameters, it's better to take the more general case instead of requiring the more specific. It makes testing easier. You could also consider adding a test here to avoid the bug that you'll discover when merging the latest changes on `master` to this branch.